controllers/shipwrightbuild_controller.go

```diff
@@ -79,6 +84,19 @@ func (r *ShipwrightBuildReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		logger.Error(err, "Retrieving ShipwrightBuild object from cache")
 		return RequeueOnError(err)
 	}
+	init := b.Status.Conditions == nil
+	if init {
+		b.Status.Conditions = make([]metav1.Condition, 0)
+		apimeta.SetStatusCondition(&b.Status.Conditions, metav1.Condition{
+			Type:    ConditionReady,
+			Status:  metav1.ConditionUnknown, // we just started trying to reconcile
+			Reason:  "Init",
+			Message: "Initializing Shipwright Operator",
+		})
+		if err := r.Client.Status().Update(ctx, b); err != nil {
+			return RequeueWithError(err)
+		}
```

Comment:
Should we requeue anyway to avoid potential update conflicts?

Reply:
done